### PR TITLE
[7.x.x] Backport new ruamel generation

### PR DESCRIPTION
**Pull request by kivistikliis**
---
### Purpose of this PR
Backports using the ruamel from sdet-tools repository, as is done for other branches.

---
### Testing status
[Weekly](https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/7.x.x%252Fyamato%252Fnew-yml/.yamato%252F_abv.yml%2523all_project_ci_weekly_2019.4/6995871/job)

[Universal](https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/7.x.x%252Fyamato%252Fnew-yml/.yamato%252Fall-universal.yml%2523PR_Universal_2019.4/6997666/job) after fixing screen resolution failures seen in the weekly above

Job outcome was compared to the [weekly from release branch](https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/7.x.x%252Frelease/.yamato%252F_abv.yml%2523all_project_ci_weekly_2019.4/6984526/job/dependencies)

---
### Comments to reviewers
We need to change the PR merge requirement from `_ABV for SRP repository - fast-2019.4` to `_ABV for SRP repository (all PR jobs) - fast-2019.4`

The PR updates OSX OpenGlCore image from `package-ci/mac:stable` to `graphics-foundation/mac-10.15:latest`

The PR also removes deprecated `packages.yml` pipelines and makes `projectcontext.yml` be the main one for package jobs

The commands setup is taken from 10.x.x/release branch

### CODE REVIEW

**Review by sophiaaar — APPROVED** [review]

File: .yamato/_projectcontext.yml [hunk 1]
```diff
@@ -245,159 +248,159 @@ publish_config_project_dry:
...
         flavor: m1.mac
     dependencies:
       -  .yamato/_projectcontext.yml#pack_all_project
-      -  .yamato/_editor_priming.yml#editor:priming:fast-2019.4:macos
+      -  .yamato/_editor_priming.yml#editor:priming:fast-2019.4:OSX
     commands:
       -  npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
       -  pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
       -  unity-downloader-cli --source-file unity_revision.txt -c editor --wait --published-only
-      -  upm-ci project test -u $(pwd)/.Editor/Unity.app/Contents/MacOS/Unity --project-path TestProjects/SRP_SmokeTest --type vetting-tests --extra-create-project-arg="-upmNoDefaultPackages"
```
Comment (theo-pnv) [inline .yamato/_projectcontext.yml:324]: I'm not sure if the upmNoDefault argument should be removed?

**Comment by theo-pnv** [comment]
Thanks, this will help a lot!
- About changing the PR merge requirements, can you do it when merging this PR?
- I'm not sure if the failure [from this job](https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/7.x.x%252Fyamato%252Fnew-yml/.yamato%252Funiversal-osx-metal.yml%2523Universal_OSX_Metal_Standalone_mono_Linear_2019.4/6997659/job) is an instability, rerunning it [here](https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/7.x.x%252Fyamato%252Fnew-yml/.yamato%252Funiversal-osx-metal.yml%2523Universal_OSX_Metal_Standalone_mono_Linear_2019.4/7000183/job) to check

File: .yamato/_projectcontext.yml (hunk 1 above)
Comment (kivistikliis) [reply .yamato/_projectcontext.yml]: Good catch, thanks, removed it!

**Review by theo-pnv — APPROVED** [review]

**Review by theo-pnv — CHANGES_REQUESTED** [review]
Seems like the minimum editor tests are failing because we're missing this folder? https://github.com/Unity-Technologies/Graphics/tree/master/.yamato/script

**Review by theo-pnv — APPROVED** [review]

**Pull request merged**